Applications/LLaMA/PyTorch/LLaMAv2.py

```diff
@@ -0,0 +1,386 @@
+# License-Identifier: "LLAMA 2 COMMUNITY LICENSE"
+# License-Details: https://github.com/facebookresearch/llama/blob/main/LICENSE    
+# Copyright (C) 2023 Meta.
```

Comment:
Are you sure that LLAMA2 community license is viral? (forcing this code as LLAMA2 community license)

Reply:
@myungjoo Thank you for pointing out the important part! I will separate the file and follow the license only for the baseline code of the model, and I will write the rest of the additional part under the Apache-2.0 license. Thanks!

Reply:
However, you may proceed with this PR and separate llama-licensed pard to another git repo later... as long as this file is not included in the product release (package file).